<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenDialog</class>
 <widget class="QDialog" name="GenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>1212</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>861</width>
     <height>791</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewTotal">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>828</y>
     <width>861</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonQuit">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>1160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>1160</y>
     <width>103</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonPrint">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>1160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOnly">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>1160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Only</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
